.github/workflows/e2e_ci.yml

```diff
@@ -95,7 +95,7 @@ jobs:
                 -DUSE_BUNDLED_LIBBPF=ON \
                 -DUSE_BUNDLED_GTEST=ON \
                 ..
-          make -j6 libsinsp_e2e_tests
+          make -j6 libsinsp_e2e_tests fake_cri
```

Comment:
I think  `fake_cri` should be automatically built as dep of `libsinsp_e2e_tests` without explicitly requiring it.

Reply:
```
add_dependencies(libsinsp_e2e_tests fake_cri)
```
Yeap, that's only a leftover